realm/realm-library/src/androidTest/java/io/realm/entities/IOSAllTypes.java

```diff
@@ -31,14 +31,12 @@
     private boolean boolCol;
     private short shortCol;
     private int intCol;
-    private long longCol;
+    private int longCol;
+    private long longLongCol;
     private float floatCol;
     private double doubleCol;
-    @Required
     private byte[] byteCol = new byte[0];
-    @Required
     private String stringCol = "";
-    @Required
```

Comment:
I just read all changes. I will review again after you will add null tests for blog and string.